index.rst

```diff
@@ -93,7 +88,34 @@ Any line that is not one of the above is invalid.
 
 EditorConfig files should be UTF-8 encoded, with LF or CRLF line separators.
 
-Additionally, EditorConfig defines the following terms:
+No inline comments
+------------------
+
+*Since version 0.15.0*
```

Comment:
I believe you can use `..version::` here

Reply:
Done!
```diff
--- a/index.rst
+++ b/index.rst
@@ -91,7 +91,7 @@ EditorConfig files should be UTF-8 encoded, with LF or CRLF line separators.
 No inline comments
 ------------------
 
-*Since version 0.15.0*
+.. versionchanged:: 0.15.0
 
 A ``;`` or ``#`` anywhere other than at the beginning of a line does *not*
 start a comment, but is part of the text of that line.  For example::
```